pa11ycrawler/spiders/edx.py

```diff
@@ -130,8 +208,10 @@ def parse_item(self, response):
         @scrapes url request_headers accessed_at page_title
         """
         # if we got redirected to a login page, then login
-        if URLObject(response.url).path in ("/login", "/register"):
-            yield self.make_login_request(response)
+        if URLObject(response.url).path == LOGIN_HTML_PATH:
```

Comment:
No, because `URLObject(response.url).path` is extracting only the path component of the URL. If there is anything in the query string, it will be available under a different property of the URLObject, and won't affect the `path` property.